.travis.yml

```diff
@@ -28,19 +28,19 @@ jobs:
     - stage: "Unit tests"
       compiler: gcc
       name: "gcc"
-      if: NOT type = cron
+      if: NOT branch = master AND type = pull_request
```

Comment:
Does the branch specifier need to be here? Should `type = pull_request` not be enough?

(for reference when `type = pull_request`, `branch` is the target/base branch (https://docs.travis-ci.com/user/conditions-v1#integration))